src/app/components/Grid/index.jsx

```diff
@@ -30,13 +30,4 @@ export const GhostGelPageGrid = styled(Grid)`
   background-color: ${C_GHOST};
 `;
 
```

Comment:
Could you also update the `gelMaxWidths` to use directly the `gel-foundations` values and remove the `group4WrapperMaxWidth` and `group5WrapperMaxWidth` constants?